src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,425 @@
+| Transmission electron microscopy              | TEM      |
+| Scanning electron microscopy                  | SEM      |
+| Micro-CT                                      | uCT      |
+| Bright-field microscopy                       | BF       |
+| Dark-field microscopy                         | DF       |
+| Phase-contrast microscopy                     | PC       |
+| Differential interference contrast microscopy | DIC      |
+| Fluorescence microscopy                       | FLUO     |
+| Confocal microscopy                           | CONF     |
+| Polarized-light microscopy                    | PLI      |
+| Coherent anti-Stokes Raman spectroscopy       | CARS     |
+| 2-photon excitation microscopy                | 2PE      |
+| Multi-photon excitation microscopy            | MPE      |
+| Super-resolution microscopy                   | SR       |
+| Nonlinear optical microscopy                  | NLO      |
+| Optical coherence tomography                  | OCT      |
+| Selective plane illumination microscopy       | SPIM     |
+
+### Filename entities
+
+In the context of Microscopy, a session ([`ses-<label>`](../99-appendices/09-entities.md#ses))
+can refer to all the acquisitions between the start and the end of an imaging experiment
+for ex vivo imaging or a subject lab visit for biopsy procedure and/or in vivo imaging.
+
+The [`sample-<label>`](../99-appendices/09-entities.md#sample) entity is REQUIRED for
+Microscopy data and is used to distinguish between different samples from the same subject.
+The label MUST be unique per subject and is RECOMMENDED to be unique throughout the dataset.
+
+<!--- The following example will be generated automatically with macros after community review. -->
+For example: Three brain slices (`sample-01` to `sample-03`) extracted from subject `sub-01`,
+imaged by scanning electron microscopy (SEM) in PNG format
+```Text
+sub-01/
+    microscopy/
+        sub-01_sample-01_SEM.png
+        sub-01_sample-02_SEM.png
+        sub-01_sample-03_SEM.png
+        sub-01_SEM.json
+```
+In this example, the JSON metadata is common for all samples of `sub-01`.
+JSON metadata may be defined per subject or per sample as appropriate, as per the
+[inheritance principle](../02-common-principles.md#the-inheritance-principle).
+
+The [`acq-<label>`](../99-appendices/09-entities.md#acq) entity corresponds to a custom label that
+MAY be used to distinguish a different set of parameters used for acquiring the same modality.
+For example, two images of the same sample acquired by bright-field microscopy (BF) in PNG format at
+different magnification of 40x and 60x.
+In such case two files could have the following names: `sub-01_sample-01_acq-40x_BF.png` and
+`sub-01_sample-01_acq-60x_BF.png`, however the user is free to choose any other label as long as
+they are consistent across subjects and sessions.
+
+The [`stain-<label>`](../99-appendices/09-entities.md#stain) entity can be used to distinguish
```

Comment:
"can be used" -> "MAY be used" to make the description consistent with the previous one for `acq`

Reply:
Thank you, I also corrected 2 other instances where "can" was used instead oy "MAY" or "MUST".